layouts/partials/latest.html

```diff
@@ -1,5 +1,5 @@
 <section id="latest">
-  <h2 class="content-center mb2">The Latest</h2>
+  <h2 class="content-center mb2">News and More</h2>
```

Comment:
I can haz ampersand, so it feels more title-y and takes up less space?
`News &amp; More`
Not passionate about this change. Just a thought.